content/blogposts/2022/release-3.43.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Sourcegraph 3.43 release"
+publishDate: 2022-08-22T10:00-07:00
+description: "Sourcegraph 3.43 introduces several new features for admins, including a newly redesigned user management page as well as annual forecasting for admin analytics."
+tags: [blog, release]
+slug: "release/3.43"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.43/sourcegraph-3-43.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.43/sourcegraph-3-43.png
+changelogItems:
+  - description: 'Search queries with patterns surrounded by /.../ will now be interpreted as regular expressions. Existing search links or code monitors are unaffected. In the rare event where older links rely on the literal meaning of /.../, the string will be automatically quoted it in a content filter, preserving the original meaning. If you happen to use an existing older link and want /.../ to work as a regular expression, add patterntype:standard to the query. New queries and code monitors will interpret /.../ as regular expressions.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/38141
+    category: Search
+  - description: 'The experimental dependencies search feature has been removed, including the repo:deps(...) search predicate and the site configuration options codeIntelLockfileIndexing.enabled and experimentalFeatures.dependenciesSearch.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/39742
+    category: Search
+  - description: 'Added the ROCKSKIP_MIN_REPO_SIZE_MB config option. Rockskip is automatically used for repositories over the set size (which defaults to 1GB).'
```

Comment:
Clarify why this is important:

This makes it easier to set up Rockskip because you no longer have to explicitly list the repositories in ROCKSKIP_REPOS.